src/control/controlbehavior.h

```diff
@@ -21,12 +21,18 @@ class ControlNumericBehavior {
     virtual double valueToMidiParameter(double dValue);
     virtual void setValueFromMidiParameter(MidiOpCode o, double dParam,
                                            ControlDoublePrivate* pControl);
+
+    // Parameter value where the control has a neutral effect
+    // or no effect. For example, for ControlAudioTaperPotBehavior this
+    // represents a knob position between 0 and 1 where
+    // the gain is 1 (0dB).
+    virtual double neutralParameter();
```

Comment:
Ha, Ok. The volume slider have their 0dB value at the very top. This will reverse the highlighted scale, I would expect that there is no scale highlighted a -∞. Am I wrong?